docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.9'
+services:
+  stargate:
+    image: 'stargateio/stargate-3_11:v1.0.29'
+    ports:
+      - '8080:8080'
+      - '8081:8081'
+      - '8082:8082'
+      - '127.0.0.1:9042:9042'
+    environment:
+      - CLUSTER_NAME=stargate
+      - CLUSTER_VERSION=3.11
+      - DEVELOPER_MODE=true
+  eddiehub-api:
+    image: ghcr.io/eddiehubcommunity/api:latest
+    ports:
+        - '3000:3000'
+    depends_on:
+        - stargate
```

Comment:
oh you used `depends_on` already 🎉 - I think the API will wait for `stargate` to be ready https://docs.docker.com/compose/startup-order/